app/helpers/application_helper.rb

```diff
@@ -34,4 +34,8 @@ def ranking_css(rank, rank_col: false)
       strong: rank > 3 && rank_col
     }
   end
+
+  def contributor_ids
+    User.contributors.pluck(:uid)
```

Comment:
This will cause the query to be run for each line in the scoreboard, that's why it was cached as a variable on the controller side. 
There's probably a better way to do it tbh but it avoids that performance issue.

Reply:
@Aquaj true, let me then simplify it by using the constant directly 😁